### PR TITLE
Allow to stop playback

### LINKED ISSUE #9: exit / stop playback from app
Add an `X` or square (stop) button to "quit" mpv. This will send the `stop` command which will stop playback and close the video, but the player will remain in an idle state (ready for a `loadfile`)

### PR DESCRIPTION
**Pull request by marcopus**
Closes #9 

This adds a stop button that let playback stop in mpv.

image

### CODE REVIEW

File: MpvRemote/app/src/main/java/miccah/mpvremote/MainActivity.java [hunk 1]
```diff
@@ -229,6 +229,9 @@ public void playPauseButton(View view) {
         BackgroundImageButton button = (BackgroundImageButton)view;
         button.sendCommand("pause", "state", !button.getState());
     }
+    public void stopButton(View view) {
+        sendCommand(null, "stop");
```
Comment (mcastorina) [inline MpvRemote/app/src/main/java/miccah/mpvremote/MainActivity.java:233]: I need to look deeper, but I *think* this "stop" command is used in the context of fast forward / rewind to allow for the press and hold style of button.

When the user first presses the button, it'll send the "repeat" command to the server. When the user releases the button, it sends the "stop" command to the server.

**Review by mcastorina — COMMENTED** [review]
Hey, thanks for your contribution, I think this'll be a great addition! When I have time I will checkout the code to test it.

At first glance, I have a few concerns though:

* I'm not certain `sendCommand(null, "stop")` does what we want it to do
* I would prefer to use a standardized stop button instead of including a PNG

File: MpvRemote/app/src/main/java/miccah/mpvremote/MainActivity.java (hunk 1 above)
Comment (marcopus) [reply MpvRemote/app/src/main/java/miccah/mpvremote/MainActivity.java:233]: I double-checked this and indeed it calls self.controller.stop() which sends a "stop" command to mpv.

Regarding the button icon, there was no stop button in the collection so I downloaed one from the same family

**Review by mcastorina — APPROVED** [review]
Works great, thanks!

**Pull request merged**